js/webpack/libraries.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
     // library
     'inject': ['./inject.js'],
     'web3': ['./web3.js'],
-    'parity': ['./parity.js']
+    'parity': ['./library.parity.js']
```

Comment:
If I don't confuse things, we can move `src/parity.js` to `npm/parity/src/index.js` and rename `src/library.parity.js` to `src/parity.js`.